src/wald_wolfowitz.jl

```diff
@@ -12,10 +12,10 @@ end
 testname(::WaldWolfowitzTest) = "Wald-Wolfowitz Test"
 population_param_of_interest(x::WaldWolfowitzTest) = ("Number of runs", x.μ, x.nruns) # parameter of interest: name, value under h0, point estimate
 default_tail(::WaldWolfowitzTest) = :both
-pvalue(test::WaldWolfowitzTest; tail=:both) = pvalue(Normal(0.0, 1.0), test.z; tail=tail)
+pvalue(test::WaldWolfowitzTest; tail = :both) = pvalue(Normal(0.0, 1.0), test.z; tail = tail)
```

Comment:
Please revert the spacing changes: AFAICT we don't use spaces like this in other files.